### PR TITLE
fix(curriculum): clarify steps wording hanoi puzzle

### PR DESCRIPTION
**Pull request by 1jds**
Checklist:

- [ x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ x ] My pull request targets the `main` branch of freeCodeCamp.
- [ x ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

I have made three changes to the wording for steps in the Python Hanoi Tower sequence. I believe that these will add a little more clarity for campers without changing any of the essential information in each step.

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc8ea01436383a88256d7f.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ dashedName: step-3
 
 # --description--
 
-The puzzle starts with the disks piled up on the first rod, in decreasing size. You need to populate your first list with numbers representing the various disk sizes.
+The puzzle starts with the disks piled up on the first rod, in decreasing size, with the smallest disk on top and the largest disk on the bottom. You need to populate your first list with numbers representing the various disk sizes.
 
-Instead of adding the items manually to the first list, generate a sequence of numbers from `3` to `1` by using the `range()` function and assign it to `rods['A']`.
+Instead of adding the items manually to the first list, generate a sequence of numbers counting down from `3` to `1` by using the `range()` function and assign it to `rods['A']`. Here, 3 represents the smallest disk at the top of the pile and 1 represents the largest disk at the bottom of the pile.
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc8ea01436383a88256d7f.md:12]: This is not true: `3` represents the largest disk on the bottom and `1` the smallest one on the top. Also numbers should be surrounded with backticks.

**Review by Dario-DC — CHANGES_REQUESTED** [review]
See my comment on the file

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc8ea01436383a88256d7f.md (hunk 1 above)
Comment (1jds) [reply curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc8ea01436383a88256d7f.md]: @Dario-DC Thank you for your feedback. Good pick up! I think I've addressed the requested changes properly now. Let me know what you think, when you have a moment.

**Review by majestic-owl448 — APPROVED** [review]
Thank you for improving the clarity of the instructions for these steps

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**